fixtures/groovy/admin/editSiteServerName.groovy

```diff
@@ -0,0 +1,20 @@
+import org.jahia.api.Constants
+import org.jahia.services.content.JCRTemplate
+import org.jahia.services.content.decorator.JCRSiteNode
+import org.jahia.services.sites.JahiaSite
+import org.jahia.services.sites.JahiaSitesService
+import org.jahia.services.sites.SiteCreationInfo
+import org.jahia.services.usermanager.JahiaUserManagerService
+import java.util.stream.Collectors
+
+JahiaSitesService sitesService = JahiaSitesService.getInstance();
+if (sitesService.getSiteByKey("SITEKEY") != null) {
+
+    JCRTemplate.instance.doExecuteWithSystemSessionAsUser(JahiaUserManagerService.instance.lookupUser("root").jahiaUser,
```

Comment:
Maybe you can replace by something like the following instead of doing sitesService.getSiteByKey twice

```
JCRTemplate.instance.doExecuteWithSystemSessionAsUser(JahiaUserManagerService.instance.lookupUser("root").jahiaUser,
    Constants.EDIT_WORKSPACE, Locale.ENGLISH, jcrsession -> {
    JCRSiteNode siteByKey = sitesService.getSiteByKey("SITEKEY", jcrsession)
    if (siteByKey != null) {
        siteByKey.setServerName("SERVERNAME")
        jcrsession.save()
}
return null
})
```